dev-util/hip/hip-5.1.3-r1.ebuild

```diff
@@ -4,8 +4,10 @@
 EAPI=8
 
 PYTHON_COMPAT=( python3_{8..11} )
+DOCS_BUILDER="doxygen"
+DOCS_DEPEND="media-gfx/graphviz"
 
-inherit cmake llvm prefix python-any-r1
+inherit cmake docs llvm prefix python-any-r1
```

Comment:
The convention of Gentoo is to use `doc`.

```
equery h doc
```

Reply:
Sorry, I can't get your point. FWICS there is only docs.eclass, not doc.eclass

Reply:
My mistake. Never mind.